packages/common/src/ui/components/navigation/AppNavLink/AppNavLink.tsx

```diff
@@ -91,75 +84,40 @@ export const AppNavLink: FC<AppNavLinkProps> = props => {
             role="link"
             aria-label={text}
             title={text}
-            onClick={onClick || debouncedClearHoverActive}
+            onClick={onClick}
           />
         )
       ),
     [to]
   );
 
-  const debouncedHoverActive = useDebounceCallback(
-    drawer.setHoverActive,
-    [],
-    HOVER_DEBOUNCE_TIME
-  );
-
-  const onHoverOver = () => {
-    if (expandOnHover) {
-      drawer.setHoverActive(to, true);
-      if (!drawer.isOpen) {
-        drawer.open();
-        drawer.setHoverOpen(true);
-      }
-    }
+  const expandChildren = () => {
+    drawer.setClickedNavPath(to);
   };
 
-  const onHoverOut = () => {
-    if (expandOnHover) {
-      debouncedHoverActive(to, false);
-    }
-  };
-
-  React.useEffect(() => {
-    const isActive = Object.values(drawer.hoverActive).some(active => active);
-    if (drawer.hoverOpen && !isActive) {
-      drawer.close();
-      drawer.setHoverOpen(false);
-      drawer.clearHoverActive();
-    }
-  }, [drawer.hoverActive]);
-
   return (
-    <Tooltip
-      disableHoverListener={drawer.isOpen && !expandOnHover}
-      title={text || ''}
-      onClose={onHoverOut}
-      onOpen={onHoverOver}
-      PopperProps={expandOnHover ? { style: { display: 'none' } } : {}}
-    >
-      <StyledListItem isSelected={selected} to={to}>
-        <ListItemButton
-          sx={{
-            ...getListItemCommonStyles(),
-            '&.MuiListItemButton-root:hover': {
-              backgroundColor: 'transparent',
-            },
-            '& .MuiTypography-root': {
-              textOverflow: 'ellipsis',
-              overflow: 'hidden',
-              whiteSpace: 'nowrap',
-            },
-          }}
-          disableGutters
-          component={CustomLink}
-        >
-          <ListItemIcon sx={{ minWidth: 20 }}>{icon}</ListItemIcon>
-          <Box className="navLinkText">
-            <Box width={10} />
-            <ListItemText primary={text} />
-          </Box>
-        </ListItemButton>
-      </StyledListItem>
-    </Tooltip>
+    <StyledListItem isSelected={selected} to={to}>
```

Comment:
no need for the tooltip now - the drawer expands on hover to show the full width nav items.
was useful for the hover, but that isn't required either now.